app/config/FrontendAppConfig.scala

```diff
@@ -52,6 +52,12 @@ class FrontendAppConfig @Inject()(configuration: Configuration) {
     "feedback-frontend.url")
   lazy val exitSurveyUrl = s"$exitSurveyBaseUrl/$serviceIdentifier"
 
+  lazy val webchatHelpUrl: String =
+    "<WEBCHAT URL GOES HERE>"
```

Comment:
instead you can use just `#`  i think ?  so you dont need to add any logic here, 
same for print as well

Reply:
Alright, will give that a go.
I didn't think JRS had a print link to use as reference, but turned out they just don't have a print link at the bottom of confirmation (just in the next steps segment), so I'll use their solution instead.